src/OrchardCore/Orchard.ContentManagement/DefaultContentManager.cs

```diff
@@ -371,6 +371,17 @@ public void Create(ContentItem contentItem, VersionOptions options)
             }
         }
 
+        public void Update(ContentItem contentItem)
+        {
+            var context = new UpdateContentContext(contentItem);
+
+            Handlers.Invoke(handler => handler.Updating(context), _logger);
+            Handlers.Reverse().Invoke(handler => handler.Updated(context), _logger);
+
+            _session.Save(contentItem);
+            _contentManagerSession.Store(contentItem);
```

Comment:
Not sure we need this. As i remember it would depend on how you previously create / get this content item. Normally if you have used the content manager API, it is already stored in the session. So this line may fail.

Maybe we don't need this line or we have to check if `_contentManagerSession.RecallVersionId()` return false. I need to think about this ...